### ISSUE TITLE
IContainer.DefaultTextStyle seems to be broken in 2022.8

### ISSUE THREAD

**Issue by harvinders** [QuestPDF/QuestPDF#310]
**Describe the bug**

It seems we can no longer set the `DefaultTextStyle` on the `IContainer `

I was able to set the DefaultTextStyle on the cell like below in 2022.6

```
   table.Header(header =>
   {
     header.Cell().Element(CellStyle).PaddingLeft(25).Text("Name");
 
    IContainer CellStyle(IContainer cellContainer)
    {
      return cellContainer.PaddingTop(1).DefaultTextStyle(TextStyle.Default
                                                .FontSize(10)
                                                .SemiBold()
                                                .FontColor(Colors.Blue.Medium));
     }
   });
```

However, with 2022.8, it has no impact. However, setting anything else, such as padding, is still working.
If I change it to 

```
   table.Header(header =>
   {
     header. Cell().PaddingLeft(25).Text("Name").Style(TextStyle.Default
                                                .FontSize(10)
                                                .SemiBold()
                                                .FontColor(Colors.Blue.Medium)); 
   });
```
it works, but it is very verbose if we need to set multiple cells with a style

**To Reproduce**
Please see the example above

**Comment by MarcinZiabek** [comment]
Hi 😁 Thank you for finding this bug. I have tried to reproduce it, and most surprisingly, it sometimes works, sometimes not. I will take a look at it possibly soon. Maybe we went a little bit to far with the optimization.

**Comment by MarcinZiabek** [comment]
This bug has been fixed in the 2022.8.1 release.
